### PR TITLE
Increasing ZooKeeper timeout and removing memory tuning options

### PR DESCRIPTION
**Pull request by escowles**

### CODE REVIEW

File: architecture-decisions/0003-remove-memory-tuning.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* `-XX:CMSMaxAbortablePrecleanTime=6000`
+
+Add the following option to disable throwing an OutOfMemoryError if garbage collection takes too long:
+* `-XX:-UseGCOverheadLimit`
+
+
+## Consequences
+
+* Not throwing an OutOfMemoryError may result in Java hanging instead of throwing an error, which would make
+  the problem instead of notifying us.
```
Comment (hackmastera) [inline architecture-decisions/0003-remove-memory-tuning.md]: can you clarify this first consequence? I am having trouble parsing it grammatically. would make the problem what?

Comment (escowles) [reply architecture-decisions/0003-remove-memory-tuning.md]: Sorry, I meant "hide the problem" instead of "make the problem"

Comment (hackmastera) [reply architecture-decisions/0003-remove-memory-tuning.md]: ahh that makes more sense! thanks

File: architecture-decisions/0002-increase-zookeeper-timeout.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Accepted
+
+## Context
+
+A ZooKeeper timeout last night brought several Solr servers down. Factors that might make servers unresponsive
+are garbage collection, network traffic/latency, and high amounts of disk I/O.
+
+## Decision
+
+Increase the ZooKeeper timeout from 15 seconds to 60 seconds.
```
Comment (jrgriffiniii) [inline architecture-decisions/0002-increase-zookeeper-timeout.md]: 👍

**Pull request merged**